src/data/services/auth.service.js

```diff
@@ -0,0 +1,61 @@
+import { gql } from "@apollo/client";
+
+const REGISTER_MUTATION = gql`
+    mutation createNewUser($firstName: String!, $lastName: String!, $email: String!,  $password: String!) {
+        register(firstName: $firstName, lastName: $lastName, email: $email, password: $password){
+            firstName,
+            lastName,
+            email,
+            id
+        }
+    }
+`;
+
+const LOGIN_MUTATION = gql`
+    mutation login($email: String!,  $password: String!){
+        login(email: $email, password: $password){
+            firstName,
+            lastName,
+            email,
+            id
+        } 
+  }
+`;
+
+const CURRENT_USER = gql`
+  query getCurrentUser{
+    getActiveUser{
+      email,
+      firstName
+    }
+}`;
```

Comment:
nit: maintain consistency with writing graphql code. As you've done with other blocks, keep `; on a separate line